Routing/AnnotatedRouteControllerLoader.php

```diff
@@ -50,7 +50,7 @@ protected function configureRoute(Route $route, \ReflectionClass $class, \Reflec
         // requirements (@Method)
         foreach ($this->reader->getMethodAnnotations($method) as $configuration) {
             if ($configuration instanceof Method) {
-                $route->setMethods(implode('|', $configuration->getMethods()));
+                $route->setMethods(explode('|', implode('|', $configuration->getMethods())));
```

Comment:
@xabbuh the last version..
I cloned the bundle and add https://github.com/sergiors/SensioFrameworkExtraBundleRouteAnnotation/commit/adc01bb0723805d67e9a2730ddafee8f97fa6f5c

run: `composer install` and `phpunit tests/Routing/AnnotatedRouteControllerLoaderTest.php`

the result:
```
PHPUnit 5.0.0 by Sebastian Bergmann and contributors.

..F                                                                 3 / 3 (100%)

Time: 144 ms, Memory: 13.50Mb

There was 1 failure:

1) Sensio\Bundle\FrameworkExtraBundle\Tests\Routing\AnnotatedRouteControllerLoaderTest::testLoad
Failed asserting that two arrays are equal.
--- Expected
+++ Actual
@@ @@
 Array (
-    0 => 'GET'
-    1 => 'POST'
+    0 => 'GET|POST'
 )
```